libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -109,6 +116,18 @@ export class CounterfactualList extends React.Component<
     );
   }
 
+  private getTargetFeatureName(): string | undefined {
+    return this.props.data?.feature_names_including_target[
```

Comment:
nit: maybe use a variable here, like:

const feature_names_including_target = this.props.data?.feature_names_including_target
return feature_names_including_target[feature_names_including_target.length - 1]

Reply:
Any particular reason why a variable will be useful here? It is going to be returned anyways.